artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java

```diff
@@ -623,11 +623,11 @@ public void setUseLocalTx(final Boolean localTx) {
       this.localTx = localTx;
    }
 
-   public boolean isRebalanceConnections() {
+   public Boolean isRebalanceConnections() {
```

Comment:
@clebertsuconic there is a JBEAP issue associated: https://issues.jboss.org/browse/JBEAP-11776 ; Fix is only needed for this reason. I am not sure how to handle such case.